packages/experiments/src/components/Stack/docs/StackOverview.md

```diff
@@ -0,0 +1,17 @@
+A Stack is a container-type component that abstracts the implementation of a flexbox in order to define the layout of its children components.
+
+## Stack Properties
+
+Although Stack has a number of different properties, there are three in particular that define the overall layout that the component has:
+
+- Direction: Refers to whether the stacking of children components is horizontal or vertical. By default the Stack component is vertical, but can be turned horizontal by adding the `horizontal` property when using the component.
+- Alignment: Refers to how are the children components aligned inside the container. This is controlled via the `verticalAlign` and `horizontalAlign` properties.
```

Comment:
does verticalAlign and horizontalAlign respect flex direction? You might mention that because in flex align is always across the 'cross axis', which switches based on the direction. 

If that's the case, we might want to point out that difference in behavior.

Reply:
Yeah, verticalAlign and horizontalAlign respect flex direction so that you don't have to be doing the mental math yourself. I'll add this to the overview.